### PR TITLE
{chem}[foss/2020a] TINKER v8.8.1

### PR DESCRIPTION
**Pull request by deniskristak**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/TINKER/TINKER-8.8.1-foss-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+description = """The TINKER molecular modeling software is a complete and general package for molecular mechanics
+ and dynamics, with some special features for biopolymers."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://dasher.wustl.edu/tinker/downloads/']
+sources = [SOURCELOWER_TAR_GZ]
+
+# patches = []
+
```
Comment (Micket) [inline easybuild/easyconfigs/t/TINKER/TINKER-8.8.1-foss-2020a.eb:14]:
```suggestion
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/25fb881da370cb39a7f5fe7ea92c11ce for a full test report.

**Comment by Micket** [comment]
Going in, thanks @deniskristak!

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Pull request merged**